browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchRequest.java

```diff
@@ -7,7 +7,7 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-class BrowserSwitchRequest {
+public class BrowserSwitchRequest {
```

Comment:
This needs to be public because it's now the return value of the `start` method - do we want a wrapper class for this? Like an object with Success/Failure values? Or is this ok?

Reply:
It's probably fine. From the SDK perspective, as on long as we mark browser-switch as `implementation` it won't be visible to the merchant unless they pull in browser switch on their own.

Reply:
I would be up for renaming this to something like `BrowserSwitchPendingRequest` or `PendingBrowserSwitchRequest` though. Does that makes sense or is it too much?